client/src/components/DatasetContainer/tests/__snapshots__/datasetContainer.test.tsx.snap

```diff
@@ -11,7 +11,9 @@ exports[`rendering of the DatasetContainer checks if various text fields are vis
       <div
         class="grid-container-desktop-lg"
       >
-        <div>
+        <div
+          class="undefined undefined"
```

Comment:
That the snapshot has `"undefined undefined"` are an indicator that's something wrong. I don't know if it's part of this changelist but it's something to investigate?